pkg/appfile/helm/helm.go

```diff
@@ -0,0 +1,93 @@
+package helm
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/oam-dev/kubevela/apis/core.oam.dev/v1alpha2"
+	helmapi "github.com/oam-dev/kubevela/pkg/appfile/helm/flux2apis"
+)
+
+// GenerateHelmReleaseAndHelmRepo constructs HelmRelease and HelmRepository in unstructured format
+func GenerateHelmReleaseAndHelmRepo(helmSpec *v1alpha2.Helm, svcName, appName, ns string, values map[string]interface{}) (helmRls, helmRepo *unstructured.Unstructured, err error) {
+	defaultIntervalDuration := &metav1.Duration{Duration: 5 * time.Minute}
+
+	rlsSpec := &helmapi.HelmReleaseSpec{}
```

Comment:
```suggestion
	releaseSpec := &helmapi.HelmReleaseSpec{}
```

It's better to name it clearly